tabuckner/broskichat
# Added a simple 404 page

### Project Changed


**- What I Did**
Added a simple 404 page.

**- How I Did It**
- Created Component
- Found some Royalty Free images we can use in the future. 
- Imported the Switch from React-Router
- Implemented.

**- How to Test/Verify It**
- launch FE app
- navigate to /test
- see 404

![image](https://user-images.githubusercontent.com/32392635/44372673-9f5f0400-a4aa-11e8-8fc8-c1ef4e5153fc.png)


**- An Image to Make Us Giggle (not mandatory but encouraged)**
